[MFLOG|0|2024-12-05T14:20:32.862360Z|runtime|4d0f2366-2901-418e-81d1-6b14d9ef40cf]2024-12-05 15:20:32,861 [WARNING] Retrying (Retry(total=4, connect=4, read=5, redirect=5, status=5)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x17bcb8380>, 'Connection to 54.90.216.156 timed out. (connect timeout=120)')': /api/2.0/mlflow/runs/get?run_uuid=6b2706f395e446308765bdb419a4b770&run_id=6b2706f395e446308765bdb419a4b770
[MFLOG|0|2024-12-05T14:21:52.048725Z|runtime|5631a95e-d0a0-43bd-b460-dac4329fa027]2024-12-05 15:21:52,048 [WARNING] Retrying (Retry(total=3, connect=3, read=5, redirect=5, status=5)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x16fba2db0>, 'Connection to 54.90.216.156 timed out. (connect timeout=120)')': /api/2.0/mlflow/runs/get?run_uuid=6b2706f395e446308765bdb419a4b770&run_id=6b2706f395e446308765bdb419a4b770
[MFLOG|0|2024-12-05T14:23:15.331072Z|runtime|9e8bb725-96bc-497e-9562-0c738c93cc85]2024-12-05 15:23:15,330 [WARNING] Retrying (Retry(total=2, connect=2, read=5, redirect=5, status=5)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x17bec8bf0>, 'Connection to 54.90.216.156 timed out. (connect timeout=120)')': /api/2.0/mlflow/runs/get?run_uuid=6b2706f395e446308765bdb419a4b770&run_id=6b2706f395e446308765bdb419a4b770
[MFLOG|0|2024-12-05T14:24:46.861695Z|runtime|84724740-3aac-4cf5-86e7-3167f615be72]2024-12-05 15:24:46,861 [WARNING] Retrying (Retry(total=1, connect=1, read=5, redirect=5, status=5)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x17bec8530>, 'Connection to 54.90.216.156 timed out. (connect timeout=120)')': /api/2.0/mlflow/runs/get?run_uuid=6b2706f395e446308765bdb419a4b770&run_id=6b2706f395e446308765bdb419a4b770
[MFLOG|0|2024-12-05T14:26:34.818329Z|runtime|56ca6140-01ee-4f57-abc5-997d6d6563a5]2024-12-05 15:26:34,817 [WARNING] Retrying (Retry(total=0, connect=0, read=5, redirect=5, status=5)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x17bec88c0>, 'Connection to 54.90.216.156 timed out. (connect timeout=120)')': /api/2.0/mlflow/runs/get?run_uuid=6b2706f395e446308765bdb419a4b770&run_id=6b2706f395e446308765bdb419a4b770
